src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/repository/BitbucketCloudRepository.java

```diff
@@ -84,10 +82,16 @@ public void setOwner(BitbucketCloudRepositoryOwner owner) {
         this.owner = owner;
     }
 
-    public void setUpdatedOn(String updatedOn) {
+    @SuppressFBWarnings("EI_EXPOSE_REP2")
+    public void setUpdatedOn(Date updatedOn) {
         this.updatedOn = updatedOn;
```

Comment:
much better to just clone the date, e.g. `this.updatedOn = updatedOn == null ? null : (Date)updatedOn.clone();` this findbugs report is generally worth paying attention to. Same for the getter

Reply:
Done